doc/ref_model/chapters/chapter08.rst

```diff
@@ -202,14 +202,31 @@ The API and Cloud Brokerage models are illustrated in the following diagrams:
-When the CSP is actings as:
-
--  consumer, in which case the typical consideration is total cost of ownership as the consumption is to usually to support internal business operations: BSS/OSS systems;
+The Communcations Service Provider (CSP) is both a provider and consumer of Cloud based services. When the CSP is acting as:
+-  consumer of cloud services, the typical consideration is total cost of ownership as the consumption is usually to support internal business operations: BSS/OSS systems;
 -  provider of cloud services, through operation of their own cloud or reselling of cloud services, in which case the typical consideration is margin (cost to offer services vs income received).
-
 These two stances will drive differing approaches to how a CSP would look to manage how it interacts within a Multi-Cloud environment.
 
+As a consumer of cloud services to support internal Business operations and BSS/OSS, the focus is on meeting the needs of the organisation's applications . Historically this came with the need to operate and support the organisation's infrastructure needs. The result was a split of the CIO organisation into Delivery and Operations groups. At the same time that the CIO application workloads are moving to SaaS and other Cloud Providers, the CTO Network Systems are migrating from running on custom dedicated infrastructure to run on virtualised COTS infrastructure; examples include IMS, 3GPP (4G & 5G) functions, IP Routers and Firewalls are being provided as VNFs and CNFs.  These network workloads are now also being deployed on private CSP clouds as well as public clouds.
+
+As outlined in section "8.2.2 Telco Cloud", the result is that the CSP "network" is now an interconnected set of distributed NFVi's (Cloud Infrastructure) supported by different Cloud Providers, including the CSP, and, hence, the term "Hybrid Multi-Cloud", and the need for the CSP to be able to support and utilize this interconnected cloud is both inevitable and essential.
+
+As a consumer and provider of Cloud Services, the CSP will continue to need to build and manage its own Cloud Infrastructure as well as provide: 
+
+- cloud orchestration solutions to orchestrate the utilisation of cloud services and capabilities from its own and other Cloud Providers;
+- network orchestration solutions to manage the interconnectivity across its own and other Cloud Provider networks. 
+
+The interactions for this are outlined in the "Multi-Cloud Interactions Model", however, to realise this, the CSP will need to adopt and sponsor a set of standards that are necessary to support these interactions. The identification of existing appliable standards and gaps across the interactions needs to be completed. As a first step, the following criteria for inclusion of a standard/technology is defined. These standards/technologies must:
+
+-  provide capabilities that are necessary to achieve hybrid multi-cloud vision and the multi-cloud interactions
```

Comment:
Please use only one space after the "-"

Reply:
@CsatariGergely I have done pass through and removed all bulleted lists with double white space. Thank you for feedback.